src/framework/arduino/wrapper/tle94112-platf-ino.hpp

```diff
@@ -9,7 +9,7 @@
 #ifndef TLE94112_PLATF_INO_HPP_
 #define TLE94112_PLATF_INO_HPP_
 
-#if defined(ARDUINO_AVR_UNO) || defined(XMC1100_Boot_Kit) || defined(XMC4700_Relax_Kit) /**< Arduino Uno, XMC1100 Boot Kit or XMC4700 Relax Kit */
+// Do not reduce the ussage to this boards, all other with SPI will work too

```

Comment:
This comment can be removed or? It loses sense once the guard is removed.

Reply:
Your right, I removed that.